### PR TITLE
Add flag to skip the build queue

### PR DESCRIPTION
**Pull request by stephybun**
The build queues and wait times until the build starts in TeamCity have been long as of late. This flag will allow us to put builds triggered by tctest at the top of the queue.

### CODE REVIEW

File: cmd/tctest/cli/cmds.go [hunk 1]
```diff
@@ -292,6 +297,7 @@ Complete documentation is available at https://github.com/katbyte/tctest`,
 		"queue-timeout": "",
 		"run-timeout":   "",
 		"latest":        "TCTEST_LATESTBUILD",
+		"skip-queue":  "",
```
Comment (katbyte) [inline cmd/tctest/cli/cmds.go:300]: could we add Ann environment var for this? as I'm going to set it to always be true 😬 
```suggestion
		"skip-queue":  "TCTEST_SKIP_QUEUE",
```

File: cmd/tctest/cli/cmds.go [hunk 2]
```diff
@@ -42,6 +42,7 @@ type FlagData struct {
 	Wait                WaitFlags
 	ServicePackagesMode bool
 	AllTests            bool
+	SkipQueue			bool
```
Comment (katbyte) [inline cmd/tctest/cli/cmds.go:45]: something weird is going on here 🤔

**Review by katbyte — CHANGES_REQUESTED** [review]
Thanks @stephybun! this is amazing and the most helpful - couple comments but otherwise looks great!

File: cmd/tctest/cli/cmds.go (hunk 1 above)
Comment (stephybun) [reply cmd/tctest/cli/cmds.go]: Good idea, done!

**Review by katbyte — APPROVED** [review]
Thanks @stephybun ! LGTM 🏗️

**Pull request merged**